### DISCUSSION TITLE
Desktop The BROWSE action is not supported on the current platform

### DISCUSSION THREAD

**Discussion by rdorsch** [devemux86/cruiser#294] [Q&A] [answered]
Hello,

I just installed cruiser for the first time. It is very impressive, how fast it is in displaying maps.

On my Debian 12 system, I see one glitch:

SCHWERWIEGEND: The BROWSE action is not supported on the current platform!

Certainly there are multiple browsers installed.

Is there a way to tell cruiser the path of the browser?

Many thanks
Rainer

**Comment by devemux86** [comment, marked as answer]
Hi @rdorsch,

This is handled internally by Java, it should open system's default browser.

What Java do you use?
What windows manager is on Debian?

I tested it on Debian (Cinnamon) with [OpenJDK 17 (Temurin)](https://adoptium.net/) and it opens the default browser.

Perhaps switching Java distribution (or version) may help.

**Reply by rdorsch** [reply]
Hi @devemux86 ,

many thanks for the quick reply.

I am using Debian 12 with KDE and openjdk:
```

rd@h370:~/tmp.nobackup/Andromaps$ java --version
openjdk 17.0.8 2023-07-18
OpenJDK Runtime Environment (build 17.0.8+7-Debian-1deb12u1)
OpenJDK 64-Bit Server VM (build 17.0.8+7-Debian-1deb12u1, mixed mode, sharing)
rd@h370:~/tmp.nobackup/Andromaps$ 
```

**Reply by devemux86** [reply]
@rdorsch 

This the Java by Debian? It has some problems (OpenGL, etc).

Please try with an external Java, like the [OpenJDK 17 (Temurin)](https://adoptium.net/).

**Comment by ramondevic** [comment]
I'm trying Cruiser Desktop and it doesn't show me the accents and other Spanish characters, how can I fix it?
"Añadir mapas", "Mapa en línea", "Cuadrícula", ...

Using Windows 10

Thanks !

![imatge](https://github.com/devemux86/cruiser/assets/122691401/e61792f1-ceee-441e-a5b7-33d2311571c4)

**Reply by devemux86** [reply]
@ramondevic can you try a different Java?
AdoptJDK (Temurin): https://adoptium.net

On Linux everything seems to work properly: :slightly_smiling_face: 

![Cruiser](https://github.com/devemux86/cruiser/assets/3484020/7d4b695c-456c-451e-87c6-3d1af6fc4740)

**Reply by ramondevic** [reply]
Perfect!
I installed it and now the special characters are visible.
Thanks !

**Reply by devemux86** [reply]
@ramondevic thanks for the feedback!

Can you mention what Java you had that did not work correctly?
(distribution and version)

**Reply by ramondevic** [reply]
Yes, 
It was Oracle Java 64bit Version 8 Update 381

**Comment by devemux86** [comment]
Why do so many users - who don't create maps or themes - use desktop Cruiser?

Is it because of the routing? Isn't it better to use the websites?

[BRouter website](https://brouter.de/brouter-web/) has more options for route planning on the desktop.

**Reply by rdorsch** [reply]
I was using QMapShack and OpenMTBMaps, but it was always troublesome for me to map the legend to the OSM tags, probably because they need to map the OSM data tags to the Garmin img format. The [OpenAndroMaps MTB style](https://www.openandromaps.org/kartenlegende/andromaps_mtb) looked much better there and I was looking for a good viewer on the desktop for them. Routing and exporting a gpx are certainly an additional benefit, but I could also continue to use QMapShack and routino there.

**Reply by 0709wiwiwi** [reply]
> Why do so many users - who don't create maps or themes - use desktop Cruiser?

Such a nice neat full-screen presentation is really a rarity.
For planning, it is particularly successful and easy to operate.
Offers 100% faithfull imports of .kurviger and gpx navigation tracks.
Displays waypoints as shaping and via and shows turn instructions.
_Brouter web does not offer gpx navtrack imports to continue working with it. :-((_
The only missing feature is BRouter offline. But this had already been mentioned.

**Reply by devemux86** [reply]
BRouter (offline) and BRouter profiles with their many options can only be used in [Android Cruiser](https://play.google.com/store/apps/details?id=gr.talent.cruiser).

(or in BRouter website)

**Reply by MotoUKRider** [reply]
For me, offline map/s (and rendered reasonably fast) offer me a useful way to look at stuff whether biking routes, places or just being nosey.

I'm not sure I could dedicate Cruiser Desktop to my primary route/waypoint filing system as BaseCamp offers me a (mostly) more efficient workflow.  Stitching maps together is really useful - a few of my routes around FR, BE, NL, DE benefit from Cruiser's ability to load multiple maps and not have to rely on a small overlap that BaseCamp's "Freizeitkarte" maps offer.

I like to keep my data offline, not relying on a server "in the cloud" or agreeing to my personal bits being used by 3rd-party etc. There's nothing wrong with online planning, routing, sharing as it offers a good incentive for developers to support the "product" and users know they are being taken care of as much as reasonably possible (support, features, bug fixes and such). The T&C on a particular online paid-for route service is quite eye-opening :)
